tests/testthat/test-pipe_continuation_linter.R

```diff
@@ -188,3 +188,21 @@ local({
     code_string = valid_code
   )
 })
+
+local({
+  linter <- pipe_continuation_linter()
+  pipes <- c("%>%", "%!>%", "%T>%", "%$%", "%<>%")
+  cases <- expand.grid(pipe1 = pipes, pipe2 = pipes, stringsAsFactors = FALSE)
+  cases <- within(cases, {
+    .test_name <- sprintf("(%s, %s)", pipe1, pipe2)
```

Comment:
`pipe2 = %<>%` makes no sense IINM.

Reply:
IIRC I've seen it used (possibly by mistake) before we cleaned up `%<>%` usage a while back. On balance, can't hurt.

Reply:
Is that even functional code?

Reply:
No, it works:

```r
x <- 1L
x %<>% c(2L) %<>% c(3L)
x
# [1] 1 2 3
```

Can't say I understand the mechanism for it to do so!

Reply:
Seems like the second `%<>%` is silently treated as `%>%`.

I remember looking at magrittrs implementation at some time in the past and it was one generic implementation for all the pipes which deparsed the symbols to make sense of the full pipeline at once...

Nevermind then.